db/migrate/20220107222719_add_user_to_vacancies_2.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToVacancies2 < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :vacancies, :user, index: true
```

Comment:
You can create index for that in one migration, you don't need to run migration removing column and then adding it with the index. There is no data in DB at the moment, but imagine such migrations in the prod with the real data. You'll remove **ALL** relations between users and vacancies, which can lead to a huge loss of data